cartographer/io/probability_grid_points_processor.cc

```diff
@@ -53,6 +54,29 @@ uint8 ProbabilityToColor(float probability_from_grid) {
              (mapping::kMaxProbability - mapping::kMinProbability)));
 }
 
+std::string FileExtensionFromOutputType(
+    ProbabilityGridPointsProcessor::OutputType output_type) {
+  if (output_type == ProbabilityGridPointsProcessor::OutputType::kPng) {
+    return ".png";
+  } else if (output_type == ProbabilityGridPointsProcessor::OutputType::kPb) {
+    return ".pb";
+  }
+  LOG(FATAL) << "OutputType does not exist!";
+}
+
+ProbabilityGridPointsProcessor::OutputType OutputTypeFromString(
+    const std::string& output_type) {
+  if (output_type == "png") {
+    return ProbabilityGridPointsProcessor::OutputType::kPng;
+  } else if (output_type == "pb") {
+    return ProbabilityGridPointsProcessor::OutputType::kPb;
+  } else {
+    CHECK(output_type == "png" || output_type == "pb")
```

Comment:
Why check again in this branch? Why return `kPb` if the impossible `CHECK` passed? Replace by `LOG(FATAL)`?

Reply:
You're right. I fixed it!